lib/stub_cas.rb

```diff
@@ -0,0 +1,26 @@
+require 'webmock'
+
+# Stub out HTTP requests to the remote CAS service, for testing / development.
+# If an email address is provided in the request (the typical cas), we return
+# that email address as the user's GUID.
+class StubCas
```

Comment:
You're quite right. That was it's original home actually. I moved it to `lib/` because there was a time where I was trying to use it in the development environment. I neglected to move it back after I abandoned that method.

Thanks!